### PR TITLE
hw-mgmt: infra: Extend leakage events and add infrastructure for L1 s…

### PR DESCRIPTION
**Pull request by vadimp-nvidia**
…witch systems

Extend leakage events and rules.
Extend power events.

### CODE REVIEW

**Comment by vadimp-nvidia** [comment]
RERUN

File: usr/usr/bin/hw-management.sh [hunk 1]
```diff
@@ -2130,6 +2133,38 @@ smart_switch_common()
...
+		connect_table+=(${p4300_base_connect_table[@]})
+		add_cpu_board_to_connection_table $cpu_bus_offset
+	fi
+	echo 1 > $config_path/global_wp_wait_step
+	echo 20 > $config_path/global_wp_timeout
+	echo 2 > $config_path/cpld_num
+	hotplug_fans=4
+	leakage_count=4
+	leakage_rope_count=2
+	max_tachos=4
```
Comment (sholeksandr) [inline usr/usr/bin/hw-management.sh:2149]: max_tachos=8

File: usr/usr/bin/hw-management.sh [hunk 2]
```diff
@@ -2130,6 +2133,38 @@ smart_switch_common()
...
+	hotplug_pwrs=0
+	hotplug_psus=0
```
Comment (sholeksandr) [inline usr/usr/bin/hw-management.sh:2151]: We have 2 replaceble PSU.
	hotplug_pwrs=2
	hotplug_psus=2
Or we can remove it and use default "2" PSUs

File: usr/usr/bin/hw-management.sh [hunk 3]
```diff
@@ -2130,6 +2133,38 @@ smart_switch_common()
...
+p4977_ns_specific()
+{
+	local cpu_bus_offset=18
+	if [ ! -e "$devtree_file" ]; then
+		connect_table+=(${p4300_base_connect_table[@]})
+		add_cpu_board_to_connection_table $cpu_bus_offset
+	fi
+	echo 1 > $config_path/global_wp_wait_step
+	echo 20 > $config_path/global_wp_timeout
+	echo 2 > $config_path/cpld_num
```
Comment (sholeksandr) [inline usr/usr/bin/hw-management.sh:2145]: 4 CPLD
echo 4 > $config_path/cpld_num

**Comment by vadimp-nvidia** [comment]
RERUN

**Review by sholeksandr — APPROVED** [review]

File: usr/usr/bin/hw-management.sh [hunk 4]
```diff
@@ -2312,15 +2350,25 @@ create_event_files()
...
+			check_n_init $events_path/leakage_rope"$i" 0
 		done
-		check_n_init $events_path/leakage_rope 0
 	fi
 	for ((i=0; i<health_events_count; i+=1)); do
 		check_n_init  $events_path/${l1_switch_health_events[$i]}
 	done
 	if [ $pwr_events_count -ne 0 ]; then
-		check_n_init $events_path/power_button 0
+		if { -f "$power_events_file" ]; then
```
Comment (sholeksandr) [inline usr/usr/bin/hw-management.sh:2365]: if { -f "$power_events_file" ]; then

Not correct bracket usage '{'. Should be used '[' instead.

**Comment by vadimp-nvidia** [comment]
RERUN

**Comment by vadimp-nvidia** [comment]
RERUN

**Comment by vadimp-nvidia** [comment]
RERUN

**Pull request merged**